app/tests/unit/lib/theme/widgets/SelectWidget.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import SelectWidget from "lib/theme/widgets/SelectWidget";
+
+import { render } from "@testing-library/react";
+
+describe("The SelectWidget Widget", () => {
+  it("Matches the snapshot with no default value set", () => {
+    const props: any = {
+      id: "1",
+      placeholder: "test placeholder",
+      onChange: jest.fn(),
+      label: "Search",
+      value: undefined,
+      required: false,
+      schema: {
+        anyOf: [
+          { value: 1, enum: [1], type: "number", title: "Option 1" },
+          { value: 2, enum: [2], type: "number", title: "Option 2" },
+        ],
+      },
+    };
+
+    const componentUnderTest = render(<SelectWidget {...props} />);
+
+    expect(componentUnderTest.container).toMatchSnapshot();
```

Comment:
snapshot testing is a good smoke test, but we need an actual assertion here.

Reply:
added a new test that tests the value of the actual dropdown